setup.py

```diff
@@ -23,16 +22,11 @@
     classifiers=[
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3",
-        "Operating System :: OS Independent"
+        "Operating System :: OS Independent",
     ],
-    project_urls={
-        "Bug Tracker": "https://github.com/enercoop/enda/issues"
-    },
-
+    project_urls={"Bug Tracker": "https://github.com/enercoop/enda/issues"},
     packages=find_packages(exclude=("tests",)),
     include_package_data=True,
-    python_requires='>=3.7.3',
-    install_requires=[
-        "pandas>=1.0.0"
-    ]
+    python_requires=">=3.7.3",
+    install_requires=["pandas>=1.0.0"],
```

Comment:
This should be consistent with the poetry requirements ?

Reply:
This diff is only formatting. The pyproject.toml deprecates setup.py but I left it for consistency. 

In the pyproject.toml the python version declared is the same, see https://github.com/enercoop/enda/pull/11/files/07b4f971cad67c6e560e9663564cc2423890ead4#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R25